relay-server/src/services/project_upstream.rs

```diff
@@ -401,7 +448,7 @@ impl UpstreamProjectSourceService {
                             result = result,
                         );
 
-                        channel.send(state.sanitized());
+                        channel.send(state);
```

Comment:
Yes! The upstream now returns `ProjectState` instead of `ProjectFetchState` to make it return the same data as the Redis source, sanitization now happens in the same place for Redis and Upstream in the project cache.